OpenDreamClient/Interface/DMF/IDMFProperty.cs

```diff
@@ -18,6 +18,7 @@ public interface IDMFProperty {
     public string AsJson();
     public string AsJsonDM();
     public string AsRaw();
+    public string AsSnowflake();
```

Comment:
Can we get a documentation comment on this? The name itself isn't very descriptive.